src/src/rx_main.cpp

```diff
@@ -547,13 +547,14 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
     {
         transmittingRadio = Radio.GetLastSuccessfulPacketRadio();
     }
-#if defined(Regulatory_Domain_EU_CE_2400)
-    transmittingRadio &= ChannelIsClear(transmittingRadio);   // weed out the radio(s) if channel in use
-    
+
     if (!geminiMode && transmittingRadio == SX12XX_Radio_All) // If the receiver is in diversity mode, only send TLM on a single radio.
     {
         transmittingRadio = Radio.LastPacketRSSI > Radio.LastPacketRSSI2 ? SX12XX_Radio_1 : SX12XX_Radio_2; // Pick the radio with best rf connection to the tx.
     }
+
+#if defined(Regulatory_Domain_EU_CE_2400)
+    transmittingRadio &= ChannelIsClear(transmittingRadio);   // weed out the radio(s) if channel in use
```

Comment:
The LBT `#ifdef` section should be placed before the `if (!geminiMode && transmittingRadio == SX12XX_Radio_All)` section.

As it is now Radio_ALL would be reduced to Radio_1 or Radio_2 before CCA with the opportunity for CCA to further reduce it to Radio_NONE. Changing the order will first do CCA on two radios increasing the chances of at least one radio surviving. If both radios survived CCA the following `if (!geminiMode && transmittingRadio == SX12XX_Radio_All)` will pick the better one just like ISM does.